package.json

```diff
@@ -12,6 +12,7 @@
     "chai": "3.5.0",
     "eslint": "3.0.1",
     "eslint-plugin-ubervu": "0.1.0",
+    "mugshot": "^0.2.0",
```

Comment:
Mugshot isn't being used anywhere as far as I can tell so no need to include it here.